src/components/delete-button/delete-button.jsx

```diff
@@ -1,30 +1,58 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import bindAll from 'lodash.bindall';
 import classNames from 'classnames';
 
 import styles from './delete-button.css';
 import deleteIcon from './icon--delete.svg';
 
-const DeleteButton = props => (
-    <div
-        aria-label="Delete"
-        className={classNames(
-            styles.deleteButton,
-            props.className
-        )}
-        role="button"
-        tabIndex={props.tabIndex}
-        onClick={props.onClick}
-    >
-        <div className={styles.deleteButtonVisible}>
-            <img
-                className={styles.deleteIcon}
-                src={deleteIcon}
-            />
-        </div>
-    </div>
-
-);
+class DeleteButton extends React.Component {
```

Comment:
We should add this behavior in a container, not the component.

Reply:
👍